Symfony/CS/Fixer/Contrib/CombineConsecutiveUnsetsFixer.php

```diff
@@ -83,8 +83,8 @@ public function getDescription()
      */
     public function getPriority()
     {
-        // should ran before SpacesAfterSemicolonFixer, WhitespacyLinesFixer, TrailingSpacesFixer and ExtraEmptyLinesFixer.
-        return 25;
+        // should ran before SpacesAfterSemicolonFixer, WhitespacyLinesFixer, TrailingSpacesFixer and ExtraEmptyLinesFixer and after NoEmptyStatementFixer.
+        return 24;
```

Comment:
Once I have started adding interface like `runAfter`/`runBefore` but the logic was too complex for handling every cases